stainless_extraction/src/bindings.rs

```diff
@@ -85,12 +95,14 @@ impl<'a, 'l, 'tcx> BodyExtractor<'a, 'l, 'tcx> {
 #[derive(Clone, Debug)]
 pub(super) struct DefContext<'l> {
   vars: HashMap<HirId, &'l st::Variable<'l>>,
+  pub params: Vec<&'l st::ValDef<'l>>,
```

Comment:
It's a bit weird to have both a private and public field in the same struct. Can we add a getter instead?